idd/Energy+.idd.in

```diff
@@ -52924,6 +53042,369 @@ AirConditioner:VariableRefrigerantFlow,
+       \note is VariableTemp
+  N14, \field Variable Condensing Temperature Maximum for Indoor Unit
+       \type real
+       \units C       
+       \default 46.0 
+       \note This field is used if Refrigerant Temperature Control Algorithm 
+       \note is VariableTemp
+  A6 , \field Outdoor Unit Fan Object Type
+       \type choice 
+       \key Fan:OnOff
+       \key Fan:ConstantVolume 
+       \default Fan:ConstantVolume 
+  A7 , \field Outdoor Unit Fan Object Name
+       \required-field
+       \type object-list
+       \object-list FansCVandOnOff
+  A8 , \field Outdoor Unit Evaporating Temperature Function of Superheating Curve Name    
+       \required-field   
+       \type object-list
+       \object-list QuadraticCurves
+       \object-list UniVariateTables
+  A9 , \field Outdoor Unit Condensing Temperature Function of Subcooling Curve Name    
+       \required-field   
+       \type object-list
+       \object-list QuadraticCurves	 
+       \object-list UniVariateTables  
+  N15, \field Diameter of main pipe connecting outdoor unit to indoor units
+       \note used to calculate the piping loss
+       \type real 
+       \units m
+       \minimum 0.0
+       \default 0.0254 
+  N16, \field Length of main pipe connecting outdoor unit to indoor units
+       \note used to calculate the heat loss of the main pipe
+       \type real 
+       \units m
+       \minimum 0.0
+       \default 30.0
+  N17, \field Equivalent length of main pipe connecting outdoor unit to indoor units
+       \note used to calculate the refrigerant pressure drop of the main pipe
+       \type real 
+       \units m
+       \minimum 0.0
+       \default 36.0
+  N18, \field Height difference between the outdoor unit node and 
+       \field indoor unit node of the main pipe
+       \note Difference between outdoor unit height and indoor unit height
+       \note Positive means outdoor unit is higher than indoor unit 
+       \note Negative means outdoor unit is lower than indoor unit 
+       \type real 
+       \units m
+       \default 5.0
+  N19, \field Insulation thickness of the main pipe
+       \type real 
+       \units m
+       \minimum 0.0	
+	   \default 0.02
+  N20, \field Thermal conductivity of the main pipe insulation material
+       \type real 
+       \units W/m-K
+       \minimum 0.0	
+	   \default 0.032
+  N21, \field Crankcase Heater Power per Compressor
+       \type real
+       \units W
+       \default 33.0
+       \note Enter the value of the resistive heater located in the compressor(s). This heater
+       \note is used to warm the refrigerant and oil when the compressor is off
+  N22, \field Number of Compressors
+       \type integer
+       \units dimensionless
+       \default 2
+       \note Enter the total number of compressor. This input is used only for crankcase
+       \note heater calculations.
+  N23, \field Ratio of Compressor Size to Total Compressor Capacity
```

Comment:
@rongpengzhang This field seems redundant. Isn't it just the inverse of "Number of Compressor"?